tests/pytest/test_error_2006_0032.py

```diff
@@ -0,0 +1,104 @@
+""" Automation of Shadow Utils tests """
+
+from __future__ import print_function
+import subprocess
+import os
+import pytest
+
+
+class TestShadowUtilsErrors():
+    """
+    Automation of Shadow Utils tests
+    """
+    def test_error_2006_0032(self):
+        """
+        :title: Tests if shadow-utils are
+         immune against bugs in 2006:0032
+        :id: TC#46675
+        :steps:
+          1. Creating tmp directory
+          2. Allowing '.' in usernames
```

Comment:
Since you removed the test you should also remove the comment

Reply:
done